Sign-up page with blacklist

HannaLindwall opened the pull request:
just accept plz

HannaLindwall committed:
blacklist

VictorWinberg reviewed (commented):
That is a pretty big file :O

Oscar-Rydh commented:
Coolt, var hitta du denna? ^^ Dock känns det som att vissa lösenord är legit.

HannaLindwall commented:
Jag googlade blacklist passwords och gick in på första sidan jag hittade x) Ja, jag förstår inte varför alla lösenord är där, vi kan ju inte preventa varenda korkat lösenord...

HannaLindwall committed:
random

HannaLindwall committed:
added three checks, which seemed appropriate(?)

HannaLindwall committed:
shorter blacklist

HannaLindwall committed:
correcting stuff

HannaLindwall committed:
correcting stuff

HannaLindwall committed:
correcting stuff

HannaLindwall committed:
username and name ain't being contained in password

HannaLindwall committed:
added some

HannaLindwall committed:
commented code

HannaLindwall commented:
In total, added checks, corrected some code, edited blacklist and commented code

HannaLindwall committed:
style

HannaLindwall committed:
less margin

VictorWinberg committed:
Layout update and cleanup

VictorWinberg committed:
placeholders added

VictorWinberg committed:
Layout and variable fix

VictorWinberg committed:
Blacklist updated to most common passwords

Using a filtered list from
https://github.com/danielmiessler/SecLists/blob/master/Passwords/10_million_password_list_top_1000.txt
only showing passwords with a length greater or equal to 8

VictorWinberg committed:
Blacklist only for password

VictorWinberg reviewed (commented):
I have made some changes to the PR, so I feel that I'm not suitable to approve the commit anymore ;) @Oscar-Rydh or @tovesson, wanna check it out?
